citi-onboarding/celeiro-dos-bambas
# Hotfix/nodemailer

## What I did

- I configure the Nodemailer module in application, for sender e-mails in contact-bar

## How to test

1. In terminal:
   1. `git checkout main`
   1. `git pull`
   1. `git checkout hotfix/nodemailer`
   1. `git pull`
   2. `yarn install` in `server/` and `client/` folders
   1. `yarn run develop` in `server/` folder
   1. `yarn run start` in `client/` folder
 2. Go to [http://localhost:3000/](http://localhost:3000/)
 3. Put on **email** and **message** in labels on contact-session
 4. Click in Submit message
 5. Log in the institucional gmail (credentials in Notion)
 6. Check if exists the email.
